### PR TITLE
Update usage to match README

### PR DESCRIPTION
**Pull request by ryan-roemer**
/cc @coopy @exogen

### CODE REVIEW

**Comment by chaseadamsio** [comment]
:+1:

File: lib/task.js [hunk 1]
```diff
@@ -120,7 +120,7 @@ Task.prototype.help = function (callback) {
   }
 
   log.info("help",
-    "\n\n" + chalk.green.bold("Usage") + ": \n\n  builder " + action + " [flags] [task]" +
+    "\n\n" + chalk.green.bold("Usage") + ": \n\n  builder " + action + " <action> <task(s)>" +
```
Comment (coopy) [inline lib/task.js]: Should this be `<action> <flags> <task(s)>`?

The only flag is currently `--builderrc`, but still.

Comment (ryan-roemer) [inline lib/task.js]: There are many flags and conditionally different. And they can go anywhere.

See: https://github.com/FormidableLabs/builder#builder-actions

Comment (coopy) [inline lib/task.js]: Oh, `--builderrc` was the only flag listed for `builder help` when I issued that command. So leave `<flag(s)>` out of the usage then?

Comment (ryan-roemer) [inline lib/task.js]: Yeah, I think so.

**Pull request merged**